examples/template/build_spreadsheet.py

```diff
@@ -1,35 +1,61 @@
 #!/usr/bin/env python
 """
-Description goes here
+A generic executor class to generate empty spreadsheets based on a set of metadata schemas.
 """
-__author__ = "jupp"
-__license__ = "Apache 2.0"
-__date__ = "08/05/2018"
 
-from ingest.template.spreadsheet_builder import SpreadsheetBuilder
+import sys
+from argparse import ArgumentParser
 
-# build a spreadsheet given new dev schemas
-dev_schemas = [
-    "http://schema.dev.data.humancellatlas.org/type/project/latest/project",
-    "http://schema.dev.data.humancellatlas.org/type/biomaterial/latest/cell_suspension",
-    "http://schema.dev.data.humancellatlas.org/type/biomaterial/latest/specimen_from_organism",
-    "http://schema.dev.data.humancellatlas.org/type/biomaterial/latest/donor_organism",
-    "http://schema.dev.data.humancellatlas.org/type/file/latest/sequence_file",
-    "http://schema.dev.data.humancellatlas.org/type/protocol/biomaterial_collection/latest/dissociation_protocol",
-    "http://schema.dev.data.humancellatlas.org/type/protocol/biomaterial_collection/latest/enrichment_protocol",
-    "http://schema.dev.data.humancellatlas.org/type/protocol/sequencing/latest/library_preparation_protocol",
-    "http://schema.dev.data.humancellatlas.org/type/protocol/sequencing/latest/sequencing_protocol",
-]
+sys.path.insert(0, '/Users/arathimani/CZI/ingest-client/')  # noqa
```

Comment:
This seems like a leftover from debugging, please remove.

Reply:
Done.